tools/gen_addon_readme.py

```diff
@@ -285,8 +285,10 @@ def gen_one_addon_index(readme_filename):
               help="git commit changes to README.rst, if any.")
 @click.option('--gen-html/--no-gen-html', default=True,
               help="Generate index html file.")
-def gen_addon_readme(
-        org_name, repo_name, branch, addon_dirs, addons_dir, commit, gen_html):
+@click.option('--template-filename', default=False,
```

Comment:
I'm not sure if click can infer the default from the method signature, but if so:

```suggestion
@click.option('--template-filename',
```